Movie/Movie/CinemaListScreen/View/CinemaListViewController.swift

```diff
@@ -0,0 +1,68 @@
+// CinemaListViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Экран со списком кинофильмов.
+final class CinemaListViewController: UIViewController {
+
+    // MARK: - Public properties
+
+    var viewModel: CinemaListScreenViewModel!
+    var internalView: CinemaListInternalView!
+    var toDescriptionScreen: ((Cinema) -> ())?
+
+    // MARK: - Private properties
+
+    private var actionHandler: TapAction?
+
+    // MARK: - life cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureTapAction()
+        configureInternalView()
+        configFetchImage()
+        configGetPopular()
+        getRequest()
+    }
```

Comment:
```suggestion
    }
```
Тяжелый ViewDidLoad лучше в отдельный метод все вынести и его вызвать